src/page/ConfigPageLayout/tabs/TerraformTab.tsx

```diff
@@ -0,0 +1,116 @@
+  const styles = useStyles2(getStyles);
+  useEffect(() => {
+    reportEvent(FaroEvent.SHOW_TERRAFORM_CONFIG);
+  }, []);
+
+  if (isLoading) {
+    return <ConfigContent loading={isLoading} title="Terraform config" />;
+  }
+
+  return (
+    <ConfigContent title="Terraform config">
+      {error && <Alert title={error.message} />}
+      <p>
+        You can manage Synthetic monitoring checks using Terraform as well as export your current checks as
+        configuration.
+      </p>
+
+      <ConfigContent.Section title="Prerequisites">
+        <div>
+          <TextLink href="https://grafana.com/docs/grafana/latest/administration/service-accounts/" external>
+            Grafana API key
+          </TextLink>
+        </div>
+
+        <div>
+          <TextLink href={`${generateRoutePath(ROUTES.Config)}/access-tokens`}>
+            Synthetic monitoring access token
+          </TextLink>
+        </div>
+      </ConfigContent.Section>
+
+      <ConfigContent.Section title="Exported config">
+        <Alert title="Terraform and JSON" severity="info">
+          The exported config is using{' '}
+          <a href="https://www.terraform.io/docs/language/syntax/json.html">Terraform JSON syntax</a>. You can place
+          this config in a file with a <code>tf.json</code> extension and import as a module. See the{' '}
+          <TextLink href="https://registry.terraform.io/providers/grafana/grafana/latest/docs" external={true}>
+            Terraform provider docs
+          </TextLink>{' '}
+          for more details.
+        </Alert>
+        <Text element="span" color="secondary">
+          Replace{' '}
+          <TextLink href="https://grafana.com/docs/grafana/latest/administration/service-accounts/" external>
+            <strong className={styles.codeLink}>&lt;GRAFANA_SERVICE_TOKEN&gt;</strong>
+          </TextLink>{' '}
+          and{' '}
+          <TextLink href={`${generateRoutePath(ROUTES.Config)}/access-tokens`}>
+            <strong className={styles.codeLink}>&lt;SM_ACCESS_TOKEN&gt;</strong>
+          </TextLink>
+          , with their respective value.
+        </Text>
+        <Clipboard
+          highlight={['<GRAFANA_SERVICE_TOKEN>', '<SM_ACCESS_TOKEN>']}
+          content={JSON.stringify(config, null, 2)}
+          className={styles.clipboard}
+          isCode
+        />
+      </ConfigContent.Section>
+
+      {checkCommands && (
+        <ConfigContent.Section title="Import existing checks into Terraform">
+          <Clipboard content={checkCommands.join(' && \\\n')} className={styles.clipboard} isCode />
+        </ConfigContent.Section>
+      )}
+
+      {probeCommands && (
+        <ConfigContent.Section title="Import custom probes into Terraform">
+          <Text element="span" color="secondary">
+            Replace{' '}
+            <TextLink href={`${generateRoutePath(ROUTES.Config)}/access-tokens`}>
+              <strong className={styles.codeLink}>&lt;PROBE_ACCESS_TOKEN&gt;</strong>
```

Comment:
```suggestion
              <strong className={styles.codeLink}>{'PROBE_ACCESS_TOKEN'}</strong>
```

Reply:
I take it as you mean `<strong className={styles.codeLink}>{'<PROBE_ACCESS_TOKEN>'}</strong>` ;)

Reply:
Oops